### PR TITLE
Minor Metadata Fixes

### PR DESCRIPTION
**Pull request by englerj**
- Restricted visibility of the Request Logs service to none. 
- Fixed a minor issue where the Request Logs debug link would not work when debugging on a local IIS server.

### CODE REVIEW

File: src/ServiceStack/RequestLogsFeature.cs [hunk 1]
```diff
@@ -96,7 +96,7 @@ public void Register(IAppHost appHost)
             }
 
             appHost.GetPlugin<MetadataFeature>()
-                .AddDebugLink(AtRestPath, "Request Logs");
+                .AddDebugLink(AtRestPath.TrimStart('/'), "Request Logs");
         }
```
Comment (mythz) [inline src/ServiceStack/RequestLogsFeature.cs]: The issue should be fixed in `.AddDebugLink()` or when link is displayed, but not here. 
What's the url that gets displayed?

Comment (englerj) [inline src/ServiceStack/RequestLogsFeature.cs]: I think I should have added just a bit more to that comment. This issue is only present when debugging on a local IIS server and the website is under a virtual directory.
In my case, I have a DemoApp project with a URL of _http://localhost/DemoApp/_.
The metadata and everything else comes up correctly at _http://localhost/DemoApp/*_
However, the request logs link is _http://localhost/requestlogs_ when it should be _http://localhost/DemoApp/requestlogs_

Comment (englerj) [inline src/ServiceStack/RequestLogsFeature.cs]: I think I should have added just a bit more to that comment. This issue is only present when debugging on a local IIS server and the website is under a virtual directory.
In my case, I have a DemoApp project with a URL of _http://localhost/DemoApp/_.
The metadata and everything else comes up correctly at _http://localhost/DemoApp/*_
However, the request logs link is _http://localhost/requestlogs_ when it should be _http://localhost/DemoApp/requestlogs_

**Comment by mythz** [comment]
I've added a complete fix for the debug/plugin links so it works in all different hosting scenarios in this commit: https://github.com/ServiceStack/ServiceStack/commit/c17323cc532239daff432d43cfa0e85d5b3e9920

**Comment by englerj** [comment]
That fixes the issue I was having. Thanks for the update!

**Pull request closed without merging**